test/base.js

```diff
@@ -81,8 +82,9 @@ describe('base config', () => {
             }
         `);
 
-        const { messages } = report.results[0];
-        assert.equal(messages.length, 1);
+        const { messages } = results[0];
+        assert.equal(messages.length, 2);
         assert.equal(messages[0].ruleId, '@lwc/lwc/no-unexpected-wire-adapter-usages');
+        assert.equal(messages[1].ruleId, '@lwc/lwc/valid-wire');
```

Comment:
Another spot where a rule was apparently added.

Reply:
This is a bug after the eslint@8 update, after investigating locally, the `@lwc/lwc/valid-wire` reports the following error:

```
"@wire" decorators can only be applied to class field and methods.
```

Support for class fields was added in eslint@8. Babel and ESTree AST use different ASTNode names to represent class fields. I am suspecting that when using eslint@8, the babel parser transforms its class fields AST nodes into the new ESTree class field shape. This transformation is causing the `@lwc/lwc/valid-wire` rule to fail.

Reply:
Fixed now with the new version of `eslint-plugin-lwc`.